File: leetcode-algorithms/035. Search Insert Position/solution.cpp
#include <iostream>
#include <vector>

using namespace std;


int searchInsert(vector<int>& nums, int target) {
	 int res = 0;
	 for(auto v:nums)
	 {
	 	if(v<target)
	 		res++;
	 }    
	 return res;
}

int main()
{
	vector<int> nums={1,3,5,6};
	int target = 0;
	int res = searchInsert(nums, target);
	cout<<res<<endl;
	return 0;
}

File: leetcode-algorithms/035. Search Insert Position/solution.py
def searchInsert(nums, target):
	"""
	:type nums: List[int]
	:type target: int
	:rtype: int
	"""
	return len([x for x in nums if x<target])

if __name__ == '__main__':
	nums = [1,3,5,6]
	target = 7

	res = searchInsert(nums, target)

	print(res)